<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="jslave-traditional-v2-10-25-236-95" name="com.mypurecloud.sdk.v2.ApiClientRetryTest" tests="12" failures="0" timestamp="16 Jan 2021 04:24:58 GMT" time="69.353" errors="0">
  <testcase name="shouldRetryTestWith_504_And_No_RetryAfter_Header" time="3.002" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="shouldRetryTestWith_429" time="1.002" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="invokeTestWith_503" time="40.035" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="invokeTestWith_429_And_No_MaxRetryTime" time="0.036" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="invokeTestWith_504" time="2.055" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="shouldRetryTestWith_502_And_0_MaxRetryTime" time="0.001" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="invokeTestWith_504_No_MaxRetryTime" time="0.031" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="invokeTestWith_502" time="13.036" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="invokeTestWith_429" time="6.062" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="shouldRetryTestWith_502" time="1.001" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="shouldRetryTestWith_503_And_RetryConfig" time="3.090" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="shouldRetryTestWith_200" time="0.002" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
</testsuite> <!-- com.mypurecloud.sdk.v2.ApiClientRetryTest -->
